src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -20,47 +26,79 @@ const getTitle = (key, t) =>
   Object.values(SIGNING_METHODS).find(item => item.key === key)?.title
     ?? t('Method not found');
 
+const getRequestTransaction = (request) => {
+  const { payload, schema } = request.request.params;
+  return decodeTransaction(Buffer.from(payload, 'hex'), schema);
+};
+
 // eslint-disable-next-line max-statements
 const RequestSummary = ({ nextStep }) => {
   const { t } = useTranslation();
+  const { getAccountByAddress } = useAccounts();
   const { events } = useContext(ConnectionContext);
   const [request, setRequest] = useState(null);
+  const [transaction, setTransaction] = useState(null);
+  const [senderAccount, setSenderAccount] = useState(null);
   const { session } = useSession();
+  useDeprecatedAccount(senderAccount);
```

Comment:
we cant handle transactions from difrent accounts, in this case you need to set current account with this sender account, its way bigger than just a deprecated account

Reply:
The whole point of `useDeprecatedAccount` was to help us sign transactions for any given account. This was requested to handle conditions in which the signature is requested for an account other than the current account. 
Since we allow the users to create connections with several accounts, we can not predict which account will be used for the request, hence we have to support this feature. 
Unfortunately at the time of reviewing the PR, I did not notice that `useDeprecatedAccount` missed this essential feature.